incident/templates/incident/topic_page.html

```diff
@@ -0,0 +1,43 @@
+{% extends "base.html" %}
+
+{% load wagtailcore_tags %}
+{% load richtext_inline from common_tags %}
+
+{% block main %}
+	<h1 class="article-title">{{ page.title }}</h1>
+
+	<article class="topicpage">
+		{% if page.description %}
+			{{ page.description|richtext_inline }}
+		{% endif %}
+
+		<section class="topicpage__inner">
+			{% include 'common/_video_or_image.html' with image=page.photo caption=page.photo_caption %}
+		</section>
+
+		<div class="topicpage-body">
+		<section class="topicpage-body__content">
+			{% for block in page.content %}
+				<div class="topicpage__block">{% include_block block %}</div>
+			{% endfor %}
+
+			<div class="topicpage__block">
+				<a class="btn btn-secondary" href="{% pageurl page.incident_index_page %}?tags={{ page.incident_tag.pk }}">
```

Comment:
@chigby 

The tag links elsewhere in the site (such as on home page incident cards, and incident listings in the incident database page) use the format:

```
?tags=protest
```

For consistency, maybe this should be amended to:

```
?tags={{ page.incident_tag.title }}"
```

Reply:
Good point! I'll make this change.